tests/Controller/EnMarche/AssessorManagerControllerTest.php

```diff
@@ -0,0 +1,320 @@
+<?php
+
+namespace Tests\AppBundle\Controller\EnMarche;
+
+use Symfony\Component\DomCrawler\Crawler;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Response;
+use Tests\AppBundle\Controller\ControllerTestTrait;
+use Liip\FunctionalTestBundle\Test\WebTestCase;
+
+/**
+ * @group functional
+ * @group assessor
+ */
+class AssessorManagerControllerTest extends WebTestCase
+{
+    use ControllerTestTrait;
+
+    private const ASSESSOR_MANAGER_EMAIL = 'commissaire.biales@example.fr';
+    private const SUBJECT_REQUEST = 'demandes? d\'assesseur';
```

Comment:
Why a `?` in `'demandes? d\'assesseur'` ? is it needed ?

Reply:
nop, removed, it was for plurelization